.changeset/late-chefs-swim.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
what's with the filename?

Reply:
Looks like one of those random generators that create memorable names by using dictionary words like for initial project names etc.

Reply:
This is temporary file, which is used to track changesets, will be removed when package is published and content will be added to CHANGELOG.md 
Example: https://github.com/grafbase/grafbase/pull/143/files